src/main/java/org/rust/lang/core/psi/RustTokenElementTypes.java

```diff
@@ -203,6 +203,18 @@
         ANDAND
     );
 
+    @NotNull
+    TokenSet ALL_OPS = TokenSet.create(
```

Comment:
Not sure if I actually need this token set; I don't use it at all. I guess it'll be useful elsewhere?